migration/src/audio.rs

```diff
@@ -58,62 +161,100 @@ struct AudioAnnotationRow {
     word: String,
 }
 
-#[non_exhaustive]
-struct AudioLayer;
-
-impl AudioLayer {
-    pub const UNLABELLED: &'static str = "";
-    pub const DOCUMENT: &'static str = "Document";
-    pub const WORD: &'static str = "Syllabary Source";
-}
-
-/// Audio resource information served by the DRS.
+/// Represents an audio resource and its annotations from an online source.
+///
+/// At this moment, each [AudioRes] _must_ have annotations present.
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct AudioRes {
-    /// The URL for a DRS audio resource
+    /// The URL associated with an audio resource.
+    ///
+    /// Currently, this URL will be from either the DRS or an S3 bucket.
     audio_url: String,
-    /// The raw text for a file marking words within a recording
+
+    /// Text holding start and end times for words contained within an audio file
```

Comment:
based on the example it looks like the word itself is also included?

Reply:
The text for the word is included in this type (represented by AudioAnnotationRow's "word" field), but is ultimately not used in the creation of an AudioSlice. So, the word data basically gets tossed out for now.